eng/pipelines/templates/stages/cosmos-sdk-client.yml

```diff
@@ -6,122 +6,127 @@ resources:
       ref: refs/tags/azure-sdk-build-tools_20211215.1
 
 parameters:
-- name: Artifacts
-  type: object
-  default: []
-- name: AdditionalModules
-  type: object
-  default: []
-- name: ServiceDirectory
-  type: string
-  default: not-specified
-- name: SDKType
-  type: string
-  default: not-specified
-- name: TargetDocRepoName
-  type: string
-  default: azure-docs-sdk-java
+  - name: Artifacts
```

Comment:
Just to make sure, the new indentation levels done in this file aren't leading to any invalid YML, right?

Reply:
the yml works - but I'll fix the indentation level.